metafix/src/test/resources/org/metafacture/metafix/integration/method/fromJson/toJson/lookupSimpleWithInternalMapDeleteNotMatchingLiteral/test.fix

```diff
@@ -0,0 +1,9 @@
+put_map("map",
+  "dog":"mammal",
+  "cat":"mammal",
+  "parrot":"bird",
+  "shark":"fish",
+  "dragon":"ficational animal",
+  "unicorn":"ficational animal")
+
+lookup("type", "map", delete: "true")
```

Comment:
Value should only be deleted if delete option is true.

Reply:
Which it is in this test.

Reply:
is default as option available or is the option just ignored? See #149

Reply:
It's ignored currently, so _accidentally_ resulting in the desired behaviour.